www/source/partials/docs/_bp-robust-supervisor-networks.md.erb

```diff
@@ -0,0 +1,91 @@
+# <a name="robust-supervisor-networks" id="robust-supervisor-networks" data-magellan-target="robust-supervisor-networks">Robust Supervisor Networks</a>
+
+Habitat Supervisors communicate amongst each other using "gossip" algorithms, which underpin the membership management, leadership election, and service discovery mechanics of Habitat. By simply being "peered" to a single existing Supervisor, a new Supervisor will gradually come to know about _all_ the Supervisors in a Habitat network. The gossip algorithm has built-in features to counteract brief network splits, but care must be taken to set up a robust Supervisor network.
+
+## The Initial Peer
+
+While a Habitat Supervisor does not need to connect with any other Supervisors in order to be useful, leveraging a network of Supervisors is really the key to unlocking the full potential of Habitat as a platform. In order to do this, a Supervisor must be given the address of at least one other Supervisor in this network when it starts up; this is known as the "initial peer problem". You might think of a Supervisor network as an exclusive members-only club; you must first know a member in order to become a member yourself.
+
+This Supervisor does _not_ know about any other Supervisors, and will (at least initially) run in complete isolation.
+```
+hab sup run
+```
+
+This Supervisor, on the other hand, will start up knowing about three other Supervisors, and will quickly establish contact with each of them. Thanks to the gossip mechanism, it will also find out about every other Supervisor those initial Supervisors know about. Similarly, every other Supervisor will discover the presence of _this_ new Supervisor.
+```
+hab sup run --peer=192.168.0.1 --peer=192.168.0.2 --peer=192.168.0.3
+```
+
+It should be noted that peering is symmetric. Even though our first Supervisor above did not start out peered with any other Supervisors, it can still become part of a Supervisor network if some other Supervisor declares it to be a peer.
+
+## Managing Membership with SWIM
+
+In order for Habitat's network functionality to work, the Supervisors must first know which other Supervisors they can communicate with. This is a problem of maintaining "membership lists", and is achieved using the membership protocol known as SWIM. As detailed above, we must first "seed" a Supervisor's membership list with at least one "peer" -- another Supervisor that it can communicate with.
```

Comment:
`SWIM` feels like it could use a link, if only for the interested. Is there a suitable one?

Reply:
https://www.cs.cornell.edu/Info/Projects/Spinglass/public_pdfs/SWIM.pdf

Reply:
Aha, I see it in the list of related links at the end of the section. 🎉